### PR TITLE
Ensure syncStateChanged is emitted

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-oc10</strong> https://drone.owncloud.com/owncloud/client/14769/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/14769/oc10/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/14769/oc10/guiReportUpload/serverlog.log

Screenshots:
- [Syncing_all_files_and_folders_from_the_server.png](https://cache.owncloud.com/public/owncloud/client/14769/oc10/guiReportUpload/screenshots/Syncing_all_files_and_folders_from_the_server.png)

</details>

File: src/gui/folder.cpp [hunk 1]
```diff
@@ -454,19 +453,21 @@ void Folder::setSyncPaused(bool paused)
...
         setSyncState(SyncResult::NotYetStarted);
     } else {
         setSyncState(SyncResult::Paused);
     }
-    emit syncPausedChanged(this, paused);
-    emit syncStateChange();
     emit canSyncChanged();
 }
 
 void Folder::setSyncState(SyncResult::Status state)
```
Comment (fmoc) [inline src/gui/folder.cpp:465]: I guess this could be renamed to `changeSyncState` to suggest that this emits a signal, too.

**Review by fmoc — CHANGES_REQUESTED** [review]

File: src/gui/folder.cpp [hunk 2]
```diff
@@ -454,19 +453,21 @@ void Folder::setSyncPaused(bool paused)
     _definition.paused = paused;
     saveToSettings();
 
+    emit syncPausedChanged(this, paused);
```
[the remark this answers is not in the document]
Comment (TheOneRing) [reply src/gui/folder.cpp:456]: I just moved the code

File: src/gui/folder.cpp (hunk 1 above)
Comment (TheOneRing) [reply src/gui/folder.cpp:465]: Thats not how Qt api is named

**Pull request merged**